tests/models/llama/test_modeling_llama.py

```diff
@@ -316,6 +316,9 @@ def setUp(self):
         self.model_tester = LlamaModelTester(self)
         self.config_tester = ConfigTester(self, config_class=LlamaConfig, hidden_size=37)
 
+    def _torch_compile_test_ckpt(self):
```

Comment:
why not use an attribute instead of a function? Should be simpler 😄

Reply:
yeah!

Reply:
Done